media-sound/qpwgraph/qpwgraph-0.7.8.ebuild

```diff
@@ -20,27 +20,16 @@ fi
 LICENSE="GPL-2"
 SLOT="0"
 
-IUSE="alsa trayicon qt6 wayland"
+IUSE="alsa trayicon wayland"
 
 BDEPEND="
-	!qt6? ( dev-qt/linguist-tools:5 )
-	qt6? ( dev-qt/qttools:6[linguist] )
+	dev-qt/qttools:6[linguist]
 "
 DEPEND="
 	media-video/pipewire:=
 	alsa? ( media-libs/alsa-lib )
-	!qt6? (
-		dev-qt/qtcore:5
-		dev-qt/qtgui:5
-		dev-qt/qtnetwork:5
-		dev-qt/qtsvg:5
-		dev-qt/qtwidgets:5
-		dev-qt/qtxml:5
-	)
-	qt6? (
-		dev-qt/qtbase:6[gui,widgets,xml,network]
-		dev-qt/qtsvg:6
-	)
+	dev-qt/qtbase:6[gui,widgets,xml,network]
```

Comment:
These USE deps should be sorted alphanumerically, and the whole `DEPEND` block should be sorted alphanumerically (with optional deps below - only alsa in this case)